docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "K8ssandra site engineer quick start"
+linkTitle: "Site engineers"
+weight: 2
+description: |
+  Kick the tires and take K8ssandra for a spin!
+---
+
+* [Upgrading K8ssandra with Ingress support]({{< relref "#configure-ingress" >}}): Enabling access to K8ssandra from outside the K8s cluster via Traefik.
+* [Accessing K8ssandra utilities]({{< relref "#access-k8ssandra-utilities" >}}): Accessing useful utilities like the Cassandra Reaper repair tool and Grafana metrics reporting.
+
+## Configure Ingress
```

Comment:
This is a pretty big topic. I would start with ingress for HTTP backed services:

- Prometheus
- Grafana
- Reaper
- Stargate auth
- Stargate REST
- Stargate Document
- Stargate Schema
- Stargate GraphQL

Separately explore native protocol / TCP ingress.

Reply:
I thought I could leave this stuff here as a quick and dirty example for the site engineer and then point to our specific Ingress instructions. I don't want to go down the rabbit hole...